### DISCUSSION TITLE
Ventura 13.7 and TP3U Plus - Perfect!

### DISCUSSION THREAD

**Discussion by chebfarid** [chris1111/Wireless-USB-Big-Sur-Adapter#336] [General]
Thank you Chris for the awesome work you are doing, génial!
Following your precise instructions I had the TP3U Plus up and working on my iMac with Ventura 13.7 in no time and it works just fine.
Thanks again, today I'll send you also a small contribute via PayPal.
Bien à toi
Farid - Milano/Italia

**Comment by chris1111** [comment]
Thank you verry mutch @chebfarid Appreciate your support.
